backend/tensorflow_backend.py

```diff
@@ -15,10 +15,12 @@
 import tensorflow as tf
 import numpy as np
 from .core import Backend
+from utils import DenseTensor
 
 
 class TensorflowBackend(Backend):
     backend_name = 'tensorflow'
+    support_sparse_format = False
```

Comment:
What's our plan for tensorflow? Or other backends?

Shouldn't we design algorithms for all the backends first (assume no per dimension sparsity storage)? Or are we specifically design an algorithm for TACO?

Reply:
For tensorflow I'm not planning to support any sparse operations. But we are not designing this specifically for TACO: if the backend supports sparse formats, our algorithms will also apply for that. CTF will be the other backend that supports sparsity.

The reason I added this support_sparse_format variable is that it can be used to check the consistency between the backend and the graph (https://github.com/LinjianMa/AutoHOOT/blob/sparse/autodiff.py#L844). If the backend doesn't support the sparse ops but we built sparse graphs, the program will just stop running.